package.json

```diff
@@ -5,22 +5,31 @@
     "typings": "./dist/typings/index.d.ts",
     "main": "dist/commonjs/index.js",
     "types": "dist/typings/index.ts",
+    "bin": {
+        "fusebox": "bin/fsbx"
+    },
     "directories": {
+        "bin": "bin",
         "test": "test",
         "modules": "modules",
         "lib": "dist/commonjs",
-        "typings": "dist/typings"
+        "typings": "dist/typings",
+        "docs": "docs"
     },
     "files": [
+        "bin",
         "src",
         "dist",
         "modules",
         "test",
         "docs"
     ],
     "scripts": {
+        "test-cli": "fusebox",
         "help": "node bin/fsbx --help",
-        "test": "TRAVIS=true ./node_modules/mocha/bin/mocha && node ./test.js",
+        "lint-fix": "eslint src/*.ts src/**/*.ts src/**/**/*.ts test/*.js test/**/*.js --fix",
```

Comment:
@Aladdin-ADD the reason it is this way is we have to be explicit about what eslint lints, as the typescript parser for eslint is not perfect. If it was, it would only need to be `eslint --fix` as the file extensions are already in the .eslint.